src/openms/thirdparty/evergreen/src/Utility/vector_ostream.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef _VECTOR_OSTREAM_HPP
+#define _VECTOR_OSTREAM_HPP
+
+#include <iostream>
```

Comment:
includes iostream, this means you cannot simply do 
```
namespace evergreen 
{
#include <Evergreen.hpp>
}
```
since it will generate a link error